app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -0,0 +1,23 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    class AuthenticationParameters
+      attr_accessor :jwt_identity, :decoded_token, :jwt_token
```

Comment:
`jwt_token` can be moved to `attr_reader` as we no longer set it outside of this class.

Reply:
please challenge the way we set `jwt_identity` & `decoded_token`. can they be set in the initialize function?